ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandServiceIT.scala

```diff
@@ -329,6 +329,9 @@ final class CommandServiceIT extends LedgerTestSuite {
     }
   })
 
+  // TODO fix this test: This test is not asserting that an interpretation error is returning a stack trace.
```

Comment:
Can we rather create a ticket for this, please? We'll have a better chance of fixing it.

Reply:
Created ticket for it